src/workerd/jsg/rtti.h

```diff
@@ -578,7 +578,11 @@ struct MemberCounter {
   template <typename Type, typename GetNamedMethod, GetNamedMethod getNamedMethod>
   inline void registerWildcardProperty() { /* not a member */ }
 
-  template <const char* name, typename Method, Method method>
+  template <const char* name,
+      typename Method,
+      Method method,
+      bool failOnIncompatibleFastApi = false,
+      bool trackFastApiUsage = false>
```

Comment:
This too. This adds a lot of noise, I guess for some testing? Is there a less-invasive way we can support testing?

Reply:
If we want to have a single method for both fast and slow paths, unfortunately this is the least amount of code change required to support this